UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -207,16 +209,17 @@ public void RestartRound()
 	{
 		if (CustomNetworkManager.Instance._isServer)
 		{
-			if (MapRotationCount <= 10)
-			{
-				CustomNetworkManager.Instance.ServerChangeScene ("Assets/scenes/OutpostDeathmatch.unity");
+			if (MapRotationCount < 10) {
+				CustomNetworkManager.Instance.ServerChangeScene (Scene[1]);
+				MapRotationCount++;
 			} 
-			else if (MapRotationCount > 10) 
+			else if (MapRotationCount >= 10) 
 			{
-				CustomNetworkManager.Instance.ServerChangeScene ("Assets/scenes/Flashlight Deathmatch.unity");
+				CustomNetworkManager.Instance.ServerChangeScene (Scene[2]);
```

Comment:
same here: your variable name is scenes, so you should writes scenes[1].

(1 = second index in array)